iml-system-test-utils/src/pdsh.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use iml_cmd::{CheckedCommandExt, CmdError};
+use std::str;
+use tokio::{fs::canonicalize, process::Command};
+
+// sed "s/\$1/2.12.4/" scripts/install_ldiskfs_no_iml.sh | pdsh -R ssh -u root -w 10.73.10.11,10.73.10.12
+
+pub async fn sed(base_path: &str) -> Result<Command, CmdError> {
+    let mut x = Command::new("sed");
+
+    let path = canonicalize(base_path).await?;
+    x.current_dir(path);
+
+    Ok(x)
+}
+
+pub async fn pdsh(hosts: &[&str], cmd: &str) -> Result<Command, CmdError> {
```

Comment:
Consider using `dshbak` or the output will be jumbled.

Reply:
Updated.